nx-X11/programs/Xserver/xkb/xkbInit.c

```diff
@@ -81,7 +74,6 @@ THE USE OR PERFORMANCE OF THIS SOFTWARE.
 #define	PHYS_LEDS	0x07
 #endif
 #endif
-#endif
 
 #define	MAX_TOC	16
 typedef struct	_SrvXkmInfo {
```

Comment:
Not related to this file: [this commit fits in here nicely](https://cgit.freedesktop.org/xorg/xserver/commit/?id=f93d10ce9bb4a6de83b561f44fb7b046def16234).